core/workspace_svg.js

```diff
@@ -91,6 +92,42 @@ Blockly.WorkspaceSvg.prototype.scrollX = 0;
 Blockly.WorkspaceSvg.prototype.scrollY = 0;
 
 /**
+ * Is zooming enabled?.
+ * @type {boolean}
+ */
+Blockly.Workspace.prototype.zooming = true;
+
+/**
+ * Current scale.
+ * @type {number}
+ */
+Blockly.Workspace.prototype.scale = 1;
+
+/**
+ * Current scale speed.
+ * @type {number}
+ */
+Blockly.Workspace.prototype.scaleSpeed = 1.2;
```

Comment:
It's hard to understand what this number indicates without some kind of unit of measurement. I'd be useful to have that in this jsdoc.